app/containers/MessageBox/ReplyPreview.js

```diff
@@ -71,19 +110,77 @@ export default class ReplyPreview extends Component {
 		close();
 	}
 
+	renderAttachment = () => {
+		const {
+			message, baseUrl, user
+		} = this.props;
+
+		if (!message.attachments) {
+			return null;
+		} else {
+			const attach = [];
+			Object.keys(message.attachments).forEach((key) => {
+				if (message.attachments[key].image_url) {
+					const img = `${ baseUrl }${ message.attachments[key].image_url }?rc_uid=${ user.id }&rc_token=${ user.token }`;
+					attach.push(
+						<FastImage
+							source={{ uri: encodeURI(img) }}
+							resizeMode={FastImage.resizeMode.cover}
+							style={styles.thumbnail}
+						/>
+					);
+				} else if (message.attachments[key].video_url) {
+					attach.push(
+						<RectButton
+							style={styles.buttonVideo}
+							activeOpacity={0.5}
+							underlayColor='#fff'
+						>
+							<CustomIcon
+								name='play'
+								size={20}
+								style={styles.playVideo}
+							/>
+						</RectButton>
+					);
+				} else if (message.attachments[key].audio_url) {
+					attach.push(
+						<RectButton
+							style={styles.buttonAudio}
+							activeOpacity={0.5}
+							underlayColor='#fff'
+						>
+							<CustomIcon
+								name='Files-audio'
+								size={20}
+								style={styles.playAudio}
+							/>
+						</RectButton>
+					);
+				}
+			});
+			return attach;
+		}
+	};
+
 	render() {
 		const {
-			message, Message_TimeFormat, customEmojis, baseUrl, username
+			message, Message_TimeFormat, customEmojis, baseUrl, user
 		} = this.props;
 		const time = moment(message.ts).format(Message_TimeFormat);
 		return (
 			<View style={styles.container}>
 				<View style={styles.messageContainer}>
 					<View style={styles.header}>
-						<Text style={styles.username}>{message.u.username}</Text>
-						<Text style={styles.time}>{time}</Text>
+						<View style={styles.text}>
+							<View style={styles.details}>
+								<Text style={styles.username}>{message.u.username}</Text>
+								<Text style={styles.time}>{time}</Text>
+							</View>
+							<Markdown msg={message.msg} customEmojis={customEmojis} baseUrl={baseUrl} username={user.username} />
```

Comment:
It should render image description below username.
<img src="https://user-images.githubusercontent.com/804994/55581395-1cb24e80-56f3-11e9-8e7e-fe1740317efe.png" width="350" />

Reply:
@diegolmello i have fixed it